packages/property-grid/package.json

```diff
@@ -92,6 +95,7 @@
     "classnames": "^2.2.5",
     "i18next": "^10.2.2",
     "react-redux": "^7.2.0",
```

Comment:
forgot to mention earlier, does this need to be a dev+peer dep now?

Reply:
react-redux does need to be a peerDep

Reply:
yep, changing in a PR, fixing all other comments too.

Reply:
fixed.